domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/SingleSurfaceForcingEffect.kt

```diff
@@ -36,8 +36,4 @@ class SingleSurfaceForcingEffect : CameraEffect(
     emptySurfaceProcessor.glExecutor,
     emptySurfaceProcessor,
     {}
-) {
```

Comment:
We should keep this. Lets add a task to make sure we release the `emptySurfaceProcessor` at the appropriate time.